src/Neogma.ts

```diff
@@ -57,6 +63,35 @@ export class Neogma {
     QueryBuilder.queryRunner = this.queryRunner;
   }
 
+  /**
+   *
+   * @param {ConnectParamsI} params - the connection params
+   * @param {ConnectOptionsI} options - additional options for the QueryRunner
+   */
+  public static fromTempDatabase = async (
+    params: ConnectParamsI,
+    options?: ConnectOptionsI,
+  ): Promise<Neogma> => {
+    const { url, username, password } = params;
+
+    const driver = neo4j.driver(
+      url,
+      neo4j.auth.basic(username, password),
+      options,
+    );
+    const database = await createTempDatabase(driver);
+    await driver.close();
+    return new Neogma({ ...params, database });
```

Comment:
I prefer adding some spacing

```suggestion
    );
  
    const database = await createTempDatabase(driver);
  
    await driver.close();
  
    return new Neogma({ ...params, database });
```